### DISCUSSION TITLE
Add feature to move around sections

### DISCUSSION THREAD

**Discussion by epicatization** [dot-cv/discussions#1] [Ideas]
Problem:
Recruiters often expect `summary` field on top of the CV. 

How to solve:
If we could move custom section (or any section) using drag and drop or simple arrows, it would solve the issue.

Or lets add optional summary section?

**Comment by kamnakis** [comment]
Thank you! A new summary section has now been added and is available in all templates.

You can check it out here https://app.dotcv.gr
